docs/how_to_guides/creating_and_editing_expectations/how_to_create_a_new_expectation_suite_using_suite_scaffold.rst

```diff
@@ -63,21 +73,23 @@ Steps
         ]
 
     To select which columns you want to scaffold Expectations on, simply uncomment them to include them.
+    
+    The next code cell shows the scaffold config, which contains the list of columns you included in the previous step, and the list of Expectations you want to generate (or not generate) for these columns. The list of included or excluded Expectations is a list of *strings* of the Expectation names, for example:
 
     .. code-block:: python
 
         scaffold_config = {
             "included_columns": included_columns,
             # "excluded_columns": [],
-            # "included_expectations": [],
+            "included_expectations": ['expect_column_values_to_not_be_null'],
             # "excluded_expectations": [],
         }
 
 3. **Generate Data Docs and review the results there**
 
     Run the next few code cells to see the scaffolded suite in Data Docs.
 
-    You can iterate on included and excluded columns and Expectations to get closer to the Suite you want.
+    Because the scaffolder is not very smart, you will want to edit this suite to tune the parameters and make any adjustments such as removing :ref:`Expectations` that don't make sense for your use case. You can iterate on included and excluded columns and Expectations to get closer to the Suite you want.
```

Comment:
Based on our pairing session yesterday @spbail I would probably recommend re-writing this to run the notebook as-is without adding `included_expectations` given that parameters not-so-great name.

Reply:
I'm going to revert the specifically "included" expectation based on @Aylr 's recommendation for now, but please feel free to continue to suggest alternate edits.